frontend/packages/dev-console/src/plugin.tsx

```diff
@@ -95,10 +95,10 @@ const plugin: Plugin<ConsumedExtensions> = [
       path: [
         '/topology/all-namespaces',
         '/topology/ns/:name',
-        '/topology/all-namespaces/graph',
-        '/topology/ns/:name/graph',
-        '/topology/all-namespaces/list',
-        '/topology/ns/:name/list',
+        '/topology/all-namespaces?view=graph',
+        '/topology/ns/:name?view=graph',
+        '/topology/all-namespaces?view=list',
+        '/topology/ns/:name?view=list',
```

Comment:
I think you can remove these paths altogether since the query params don't matter here.

Reply:
removed, only kept ones that are without the query params

Reply:
Can I presume this change is OCP 4.9+ only?

Reply:
Yes that's correct.